todo_app/tests/unit_tests/ViewModel_test.py

```diff
@@ -0,0 +1,33 @@
+from todo_app.data.ViewModel import ViewModel
+from pytest import fixture
+
+class MockItem:
```

Comment:
Reasonable here, but worth noting that you might find you regularly need to update this MockItem as your ViewModel changes; and just using the genuine ToDoItem class might save that effort. However the default constructor for ToDoItem is doing a fair amount at the moment (transforming the status) so we'd probably want to adjust that if we were going to do that. (i.e. adjust ToDoItem so that it just stores properties passed through, and whoever creates a ToDoItem has to worry about converting list ids -> status first)